libs/sdk/src/core/schema.ts

```diff
@@ -373,6 +376,10 @@ export const ConfigurationSchema = z.strictObject({
   consumer_groups: z.array(consumerGroupSchema).optional(),
   global_rules: globalRuleSchema.optional(),
   plugin_metadata: pluginMetadataSchema.optional(),
+  routes: z.array(routeSchema).optional(),
+  stream_routes: z.array(streamRouteSchema).optional(),
+  consumer_credentials: z.array(consumerCredentialSchema).optional(),
+  upstreams: z.array(upstreamSchema({ id: idSchema.optional() })).optional(),
```

Comment:
These elements do not exist in the ADC schema and users are not permitted to configure these fields here. Do not modify the core schema.
If your type system requires these fields, create another type alias.

```ts
const AnotherConfiguration = z.infer<typeof ConfigurationSchema> & {xx: xx, yy: yy};
```